index.bs

```diff
@@ -3801,7 +3801,7 @@ of running the following steps:
     : "`scheduled_report_time`"
     :: |report|'s [=aggregatable report/report time=] in seconds since the UNIX epoch, [=serialize an integer|serialized=]
     : "`version`"
-    :: A [=string=], API version.
+    :: "`0.1`"
```

Comment:
Let's add a non-normative note here indicating the circumstances in which this value needs to be bumped, and/or link to the corresponding guidance from the aggregation service.

Reply:
Added a note, thanks.